src/Menu.js

```diff
@@ -136,17 +144,31 @@ class Menu {
       // The focusout event is unreliable across browsers, operating systems, front-end test
       // frameworks, etc. so instead check whether an element that is not a descendant has received
       // a focus-changing event.
-      $(document).on('keypress click focusin', (event) => {
-        if (this.openIndex !== -1 && $element.has(event.target).length === 0) {
-          this.setActiveElement(this.items[this.openIndex].$link);
+      document.addEventListener('keydown', (event) => {
+        if (this.openIndex !== -1 && !this.element.contains(event.target)) {
```

Comment:
Since these are all the same it might be cleaner to create the function as a const and then supply as the delegate. 

```
  const foo = (event) => { some stuff happening };
  document.addEventListener('keydown', foo);
  document.addEventListener('click', foo);
  document.addEventListener('other events', foo);
```